articles/multifactor-authentication/google-auth/user-guide.md

```diff
@@ -0,0 +1,71 @@
+# How to Use Google Authenticator
+
+Google Authenticartor is a mobile app used for two-factor authentication when logging into an application, which helps create a more secure login.  With two-factor authentication you will always need your mobile device when you log in.
+
+This page will help to explain how to sign up and log in using the Google Authenticator app.
+
+## Download the Google Authenticator App
+
+### Android 
+Requirements: Must be running Android version 2.1 or later.
+
+To download the app:
+
+1. Visit [Google Play](https://play.google.com/store).
+2. Search for Google Authenticator.
+3. Download and install the application.
+
+### iOS
+Requirements: Must have iOS 5.0 or later. In addition, in order to set up the app on your iPhone using a QR code, you must have a 3G model or later.
+
+To download the app:
+
+1. Visit the [App Store](itunes.apple.com/AppStore).
+2. Search for Google Authenticator.
+3. Download and install the application.
+
+### Blackberry
+
+Requirements: Must have OS 4.5-7.0. In addition, make sure your BlackBerry device is configured for US English -- you might not be able to download Google Authenticator if your device is operating in another language.
+
+To download the app:
+
+1. Open the web browser on your BlackBerry.
+2. Visit [m.google.com/authenticator](m.google.com/authenticator).
+3. Download and install the application.
+
+## Sign Up as a New User
+
+If you do not have an existing account, you will need to sign up to create one. Click the **SIGN UP** button and enter your email and create a password.
+
+![](/media/articles/mfa/sign-up.png)
+
+You will need to download Google Authenticator app.
+
+A code will appear, and you will have five minutes to scan the code before it expires. After scanning the code, you will get a six digit code to enter. Once you enter this code, you will see a confirmation screen which has a recovery code. If for some reason you do not have your mobile device, you will need this recovery code to login. Make sure to take note of this code and keep it somewhere safe. Check the box that you have recorded the code, and then you are logged in.
+
+![](/media/articles/mfa/google-code.png)
+
+## Logging in
+
+After entering your username and password, you will be prompted for a six digit code. Open the Google Authenticator app on your mobile device to find the correct code. If you use Google Authenticator for other applications as well, make sure you are using the code for the current application. After entering the six digit code you will be logged in.
+
+## Troubleshooting
```

Comment:
What about mentioning that device clock must be sync. We recommend using automatic sync if available.